BUILD.bazel

```diff
@@ -15,6 +15,13 @@
 load("@gazelle//:def.bzl", "gazelle")
 load("@protobuf//bazel:proto_library.bzl", "proto_library")
 load("@rules_go//proto:def.bzl", "go_proto_library")
+load("@rules_buf//buf:defs.bzl", "buf_lint_test")
+
+buf_lint_test(
+    name = "federation_lint_test",
+    targets = [":federation_proto"],
+    config = "buf.yaml",
```

Comment:
I'm a little surprised that the yaml config is part of the lint test label and not e.g. configured in MODULE.bazel or some such? Perhaps they don't yet offer support for this? But it's unfortunate to have to include / not forget the buf.yaml include on every buf_lint_test.